src/views/Networking.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="networking-page"
+    >
+      <input
+        v-model="room"
+        :placeholder="$t('Networking.Placeholder')"
+      >
+      <RouterLink :to="{ name: 'conference', params: { room } }">
+        {{ $t('Networking.Start') }}
+      </RouterLink>
+    </div>
+  </Page>
+</template>
+<script>
+import Page from '../components/layout/Page.vue';
+
+export default {
+  name: 'Networking',
+  components: {
+    Page,
+  },
+  data: () => ({
+    room: '',
+  }),
+  methods: {
+    newRoom() {
+      this.$router.push({
+        name: 'networking',
+        params: {
+          room: this.room,
+        },
+      });
+    },
+  },
```

Comment:
```suggestion
```

Reply:
Already does that. Now we have nice path like `http://superhero.com/networking/:room`

Reply:
Got that. We don't need it anymore.

Reply:
Removed, thanks